### ISSUE TITLE
Siri commands for Tilt (Up vs Down)

### ISSUE THREAD

**Issue by DevMan01** [ebaauw/homebridge-sc#12]
I love this plugin, and I'm very grateful for the work you've done.

I'm sure this is an `id10t` error on my side. But, I don't know which Siri commands tell Soma to tilt the blinds upwards or downwards?

I've tried things like `Hey Siri, close the blinds upwards / downwards`. Or I've tried commands like `Hey Siri, tilt the blinds upwards / downwards.` I'm sure there is a command that Siri will recognize, but I cant seem to figure it out. 

The tl;dr is that I simply want to be able to tell Siri to close the blinds upwards or downwards. I am able to go into the Home app and tap the blinds, and they will cycle through various positions, closed (up), open, closed (down).

**Comment by ebaauw** [comment]
_Close Upwards_ is a custom characteristic that neither Siri not Home wants to understand.  Best create HomeKit scenes in Eve and recall these from Siri and / or Home.
